File: app/src/main/java/com/jonasgerdes/stoppelmap/model/shared/GeoLocation.java
package com.jonasgerdes.stoppelmap.model.shared;

import com.google.android.gms.maps.model.LatLng;

import io.realm.RealmObject;

/**
 * Created by Jonas on 08.07.2016.
 */

public class GeoLocation extends RealmObject {

    public GeoLocation() {
    }

    public GeoLocation(double lat, double lng) {
        this.lat = lat;
        this.lng = lng;
    }

    private double lat;
    private double lng;

    public LatLng toLatLng() {
        return new LatLng(lat, lng);
    }

    public double getLat() {
        return lat;
    }

    public double getLng() {
        return lng;
    }
}

File: app/src/main/java/com/jonasgerdes/stoppelmap/model/entity/GeoLocation.kt
package com.jonasgerdes.stoppelmap.model.entity

import com.google.android.gms.maps.model.LatLng
import com.google.gson.annotations.SerializedName
import io.realm.RealmObject

/**
 * @author Jonas Gerdes <dev@jonasgerdes.com>
 * @since 13.06.2017
 */
open class GeoLocation : RealmObject() {
    @SerializedName("lon")
    var longitude: Double = 0.0
    @SerializedName("lat")
    var latitude: Double = 0.0

    val latLng get() = LatLng(latitude, longitude)
}